ambry-api/src/main/java/com/github/ambry/store/MessageInfo.java

```diff
@@ -429,6 +442,7 @@ public Builder expirationTimeInMs(long expirationTimeInMs) {
      * @param isDeleted the isDeleted to set.
      * @return This builder.
      */
+    @JsonProperty("deleted")
```

Comment:
I guess by default, Jackson renames primitive boolean field by removing 'is' and It'll become "deleted". Right?
Are you intended to use @JsonProperty("isDeleted")?

Reply:
yes. it's one way or another. we either annotate on MessageInfo or MessageInfo.Builder. I don't see there is a difference.